src/components/Activities/Activity3/Game/FeaturesSlidebar.tsx

```diff
@@ -0,0 +1,129 @@
+import React, { useContext, useEffect, useState } from 'react';
+import '../../../styles/Activity3Game.scss';
+import { GameContext } from '.';
+
+interface FeatureSlidebarProps {
+  initialFeatureWeights: number[],
+  setFeatureWeights: (featureWeights: number[]) => void,
+}
+
+function FeatureSlidebar(props: FeatureSlidebarProps): JSX.Element {
+  const { goNextState, variableSelection } = useContext(GameContext);
+  const { initialFeatureWeights, setFeatureWeights } = props;
+  const [weights, setWeights] = useState(initialFeatureWeights);
+
+  useEffect(() => {
+    setFeatureWeights(weights);
+  }, [weights]);
+
+  const handleInputWeight = (index: number) => (e: React.ChangeEvent<HTMLInputElement>) => {
+    const valString: string = e.currentTarget.value.replace('%', '').trim();
+    const val: number = valString === '' ? 0 : parseInt(valString);
+    switch (index) {
+      case 0:
+        adjustWeights1(val);
+        break;
+      case 1:
+        adjustWeights2(val);
+        break;
+      case 2:
+        adjustWeights3(val);
+    }
+  };
+
+  const scrollValue = (index: number) => (e: React.WheelEvent) => {
+    let adjustment = 0;
+    if (e.deltaY < 0) { //scrolling down
+      adjustment = -1;
+    } else if (e.deltaY > 0) { //scrolling up
+      adjustment = 1;
+    }
+    switch (index) {
+      case 0:
+        adjustWeights1(weights[0] + adjustment);
+        break;
+      case 1:
+        adjustWeights2(weights[1] + adjustment);
+        break;
+      case 2:
+        adjustWeights3(100 - weights[0] - weights[1] + adjustment);
+    }
+  };
+
+  const adjustWeights1 = (val: number) => {
```

Comment:
nit: this seems like a first function of `adjustWeights`! this comment should help a bit with code reading (feel free to modify)
```suggestion
    // adjusts weights given 1st weight's percentage
  const adjustWeight1 = (val: number) => {
```

Reply:
oops i see that you have comments for 2 & 3, feel free to just apply that here maybe!